src/ZuriAC.php

```diff
@@ -166,7 +171,7 @@ public function loadChecks() : void {
 
 		// Fly
 		$this->checks[] = new \ReinfyTeam\Zuri\checks\fly\FlyA();
-		$this->checks[] = new \ReinfyTeam\Zuri\checks\fly\FlyB();
+		//$this->checks[] = new \ReinfyTeam\Zuri\checks\fly\FlyB();
```

Comment:
Why this comment?

Reply:
it is totally unreadable and not function at all
check [BedrockProtocol](https://github.com/pmmp/BedrockProtocol/blob/master/src/UpdateAdventureSettingsPacket.php)